packages/server/src/addresses/database/AddressSchema.ts

```diff
@@ -0,0 +1,62 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Exclude, Expose } from 'class-transformer';
+import { ObjectId } from 'mongodb';
+import { Document } from 'mongoose';
+
+type AddressDocument = Address & Document<ObjectId>;
+
+const ADDRESS_CONSTANTS = Object.freeze({
+  STREET_NUMBER: Object.freeze({ REGEX: /^([0-9]{1,})([A-Za-z]{0,1}$)/ }),
```

Comment:
Ten regex można uprościć do postaci:
```suggestion
  STREET_NUMBER: Object.freeze({ REGEX: /^\d+[A-Za-z]?$/ }),
```
Swoją drogą zawsze dawaj ograniczenia na długość danych albo w `regex` albo w `maxLength`. Jest to zabezpieczenie przed złośliwym działaniem użytkowników, np. [wpisaniem całej treści Pana Tadeusza](https://geek.justjoin.it/albicla-polski-odpowiednik-facebooka) :upside_down_face: